lte/gateway/c/core/oai/tasks/nas/emm/msg/AttachRequest.c

```diff
@@ -338,6 +338,15 @@ int decode_attach_request(
         decoded += 1;  // Device Properties is 1 byte
         break;
 
+      case ATTACH_REQUEST_EXTENDED_DRX_PARAMETERS_IEI:
+        // Skip this IE.
+        OAILOG_INFO(
+            LOG_NAS_EMM,
+            "EMM-MSG - Extended DRX Parameters IE in Attach Request is not "
+            "supported. Skipping this IE.");
+        decoded += 3;  // Extended DRX Parameters is 3 bytes
```

Comment:
This IE is a type 4 information element. You need to implement a `decode_extended_drx_parameter_ie` function that checks the existence of IEI (1 octet), then read the length (1 octet) and then skip the remaining value part based on its length read in the previous octet.

Reply:
you can check `decode_voice_domain_preference_and_ue_usage_setting` to see a sample.